### DISCUSSION TITLE
Set Firefox profile for selenium when using CLI

### DISCUSSION THREAD

**Discussion by scruel** [gildas-lormeau/SingleFile#811] [Q&A]
**Is your feature request related to a problem? Please describe.**
Could not pass the profile arguments to Firefox, it still uses the default profile.
single-file https://www.wikipedia.org "wikipedia.html" --back-end=webdriver-gecko --browser-headless=false --browser-args "[\"-P C:\\Users\\scruel\\AppData\\Roaming\\Mozilla\\Firefox\\Profiles\\test.default-release\"]"

**Describe the solution you'd like**
I have tried https://github.com/gildas-lormeau/SingleFile/issues/599 mentioned way to add arguments, but it still not run with profile's extensions.

**Comment by gildas-lormeau** [comment]
You must escape quotes and backslashes in the arguments string, and also separate `-P` and its value:
```
single-file 
  https://www.wikipedia.org 
  wikipedia.html 
  --back-end=webdriver-gecko 
  --browser-headless=false 
  --browser-args "[\"-P\",\"C:\\Users\\scruel\\AppData\\Roaming\\Mozilla\\Firefox\\Profiles\\test.default-release\"]"
```

**Reply by scruel** [reply]
> 
> 
> You must escape quotes and backslashes in the arguments string, and also separate `-P` and its value:
> 
> ```
> single-file 
>   https://www.wikipedia.org 
>   wikipedia.html 
>   --back-end=webdriver-gecko 
>   --browser-headless=false 
>   --browser-args "[\"-P\",\"C:\\Users\\scruel\\AppData\\Roaming\\Mozilla\\Firefox\\Profiles\\test.default-release\"]"
> ```

It's still not work, and is using the temp profile "C:\Users\scruel\AppData\Local\Temp\rust_mozprofile".

**Reply by gildas-lormeau** [reply]
Then, it might be a bug in Selenium. I verified the syntax was correct by debugging SingleFile CLI.

**Reply by scruel** [reply]
@gildas-lormeau Yep, I tried active the profile via python code, but is still not work.

**Reply by scruel** [reply]
@gildas-lormeau Sad... https://github.com/SeleniumHQ/selenium/issues/9997#issuecomment-960761423

**Reply by gildas-lormeau** [reply]
Indeed... I'm wondering if this issue also exists with playwright or puppeteer.
